options/train_options.py

```diff
@@ -53,5 +53,14 @@ def initialize(self, parser):
         #all classes are the same options
         parser.add_argument('--all_classes_as_one',action='store_true',help='if true, all classes will be considered as the same one (ie foreground vs background)')
 
+        #Adaptative pseudo augmentation using G
+        parser.add_argument('--adaptive_pseudo_augmentation', action='store_true',help='if true, G will be used as augmentation during D training adaptively to D overfitting between real and fake images')
+        parser.add_argument('--APA_target',type=float,default=0.6)
+        parser.add_argument('--APA_p',type=float,default=0.5,help='initial value of probability APA')
+        parser.add_argument('--APA_every', type=int, default=4,help='How often to perform APA adjustment?')
+        parser.add_argument('--APA_display_every', type=int, default=100,help='How often to display current p prob')
```

Comment:
To simplify options, we could simply print it out according to the main `print_env` argument.